### ISSUE TITLE
Unable to create .co.uk records

### ISSUE THREAD

**Issue by craig0990** [namecheap/go-namecheap-sdk#13]
Howdy,

Getting an error with the terraform provider:

```
2020/05/30 16:47:38 [ERROR] <root>: eval: *terraform.EvalApplyPost, err: Failed to create namecheap Record: Domain ["mydomain" "co" "uk"] does not contain SLD and TLD
```

Which appears to come from here: https://github.com/adamdecaf/namecheap/blob/master/host.go#L26

And is presumably caused by the `Split` call on line 24 above it.

I'm not too familiar with golang in general, but https://godoc.org/golang.org/x/net/publicsuffix looks like it might be a better solution for this sort of thing - unless, of course, I've misconfigured something ;)

**Comment by craig0990** [comment]
Actually I've had a crack at this and you can see the result in #14. I'm not good enough with golang to try and run the whole terraform provider locally - I assume I'd have to put everything in master, update all the import paths, and build it locally, and I'd rather see the feedback first :grin:

**Comment by ifree92** [comment]
Thanks for your patience so far.
This issue has been resolved in #24
